base/stats.go

```diff
@@ -1,13 +1,49 @@
 package base
 
-import "expvar"
+import (
+	"expvar"
+	"time"
+)
 
 var StatsExpvars *expvar.Map = expvar.NewMap("syncGateway_stats")
 var TimingExpvars SequenceTimingExpvar
+var perDbStatNames = []string{
+	"changesFeeds_active",
+	"changesFeeds_total",
+	"requests_active",
+	"requests_total",
+	"revisionCache_hits",
+	"revisionCache_misses",
+	"simpleChanges_active",
+	"simpleChanges_total",
+	"subChanges_active",
+	"subChanges_total",
+	"vectorChanges_active",
+	"vectorChanges_total"}
+var globalStatNames = append(perDbStatNames, []string{
+	"bulkApi.BulkDocsPerDocRollingMean",
+	"bulkApi.BulkDocsRollingMean",
+	"bulkApi.BulkGetPerDocRollingMean",
+	"bulkApi.BulkGetRollingMean",
+	"goroutines_highWaterMark",
+	"handler.CheckAuthRollingMean",
+	"indexReader.getChanges.Count",
+	"indexReader.getChanges.Time",
+	"indexReader.getChanges.UseCached",
+	"indexReader.getChanges.UseIndexed",
+	"indexReader.numReaders.OneShot",
+	"indexReader.numReaders.Persistent",
+	"indexReader.pollPrincipals.Count",
+	"indexReader.pollPrincipals.Time",
+	"indexReader.pollReaders.Count",
+	"indexReader.pollReaders.Time",
+	"indexReader.seqHasher.GetClockTime",
+	"indexReader.seqHasher.GetHash"}...)
```

Comment:
Expvars shouldn't need to be explicitly initialized in most cases.  Including their definitions here feels like unnecessary maintenance overhead.